src/lib/stepper/stepper-module.ts

```diff
@@ -54,6 +54,6 @@ import {MatStepperIntl} from './stepper-intl';
     MatStepHeader,
     MatStepperIcon,
   ],
-  providers: [MatStepperIntl, ErrorStateMatcher],
+  providers: [MAT_STEPPER_INTL_PROVIDER, ErrorStateMatcher],
```

Comment:
@jelbourn I would've assumed that this was already taken care of by using `providedIn: 'root'`, but based on the discussion in #12904 that doesn't seem to be the case. I wonder whether it isn't breaking down, because we were declaring the provider in here.